### DISCUSSION TITLE
Welcome to NOAAHackDays Discussions!

### DISCUSSION THREAD

**Discussion by eeholmes** [nmfs-opensci/NOAAHackDays#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by elcansaito** [comment]
Trying to replicate your video  https://www.youtube.com/watch?v=SZ0WJMHGDps, but I have some issues.

- I copied everything using the terminal and that's fine.
- I changed the name, copy the code from Minute 3:45 into my YAML, add a different picture for the logos and when I render, there is no change in the pdf. 
- The html is changed but I want the pdf and the pdf doesn't show any changes at all. 
- I got Pic 1, and If I click on the top right I got Pic2. The html is changed but the PDF doesn't change.
<img width="1027" alt="Pic1" src="https://github.com/nmfs-opensci/NOAAHackDays/assets/8972010/14e86f41-e9fa-4103-b46d-ef88933c2f6e">
<img width="1027" alt="Pic2" src="https://github.com/nmfs-opensci/NOAAHackDays/assets/8972010/276887e6-945e-46f1-924a-8a5b1f4d2a6e">


- It's strange since in your video the pdf changes automatically. I'd appreciate any help to fix it so I can use the template. Thanks in advance. 



This is my YAML:


---
title: Report
subtitle: Innovation
author:
  - name: Myself
  - name: Yourself
bibliography: references.bib
format: 
  html: default
  titlepage-pdf:
    documentclass: scrbook
    classoption: ["oneside", "open=any"]
    number-sections: true
    toc: true
    lof: true
    lot: true
    titlepage: "bg-image"
    titlepage-bg-image: "img/ringed-seal.png"
    titlepage-logo: "img/ringed-seal.png"
    titlepage-header: "The Publisher"
    titlepage-geometry: ["top=3in", "bottom=1in", "right=1in", "left=1in"]
    titlepage-footer: |
      NOAA Fisheries OpenSci\
      Tools for Open Science\
      [https://github.com/nmfs-opensci](https://github.com/nmfs-opensci)\
    titlepage-theme:
      elements: ["\\titleblock", "\\authorblock", "\\affiliationblock",  "\\logoblock", "\\vfill"]
      page-align: "left"
      title-style: "plain"
      title-fontstyle: ["large", "bfseries"]
      title-space-after: "4\\baselineskip"
      subtitle-fontstyle: ["large", "textit"]
      author-style: "superscript-with-and"
      author-fontstyle: "large"
      affiliation-style: "numbered-list-with-correspondence"
      affiliation-fontstyle: "large"
      affiliation-space-after: "0pt"
      footer-style: "plain"
      footer-fontstyle: "large"
      logo-size: "0.25\\textheight"
      logo-space-after: "2\\baselineskip"
      vrule-width: "1pt"
      bg-image-size: "0.5\\paperwidth"
      bg-image-location: "LRCorner"
    keep-tex: true
---

**Comment by eeholmes** [comment]
Hi, It sounds like your GitHub Action which rebuilds the book (PDF) is not running. See the instructions here
https://github.com/nmfs-opensci/NOAA-quarto-book?tab=readme-ov-file#github-set-up
and make sure you 1) have the gh-pages branch and 2) have GitHub Actions turned on for your repo.

**Reply by elcansaito** [reply]
Thanks a million for your fast reply. I´ll do it right away!!!
